### PR TITLE
Add FAQ section, change small errors

### PR DESCRIPTION
**Pull request by WillemGerritzen**

### CODE REVIEW

File: src/pages/onboarding.adoc [hunk 1]
```diff
@@ -25,11 +25,9 @@ Picnic introduction, your gear and set-up the office systems. In the
 afternoon, you'll spend 2-3 hours with the Customer Success team and
 meet your buddy (see below).
 * *Tuesday*: You'll have the chance to spend some time at the heart of
-our supply chain. For half a day, you'll be a shopper, working in a
-Fulfillement Center, and the other half, you'll jin a runner on a
-delivery shift. This is done to allow you to get a solid understanding
-of our supply chain and operations.
-* *Wednesday*: First work-day! You'l be properly introduced to the team
+our supply chain. For half a day, you'll tag along a runner to see what the deliveries look like for Picnic and the customers.
```
Comment (Stephan202) [inline src/pages/onboarding.adoc:28]:
```suggestion
our supply chain. For half a day, you'll tag along with a runner to see what the deliveries look like for Picnic and the customers.
```

File: src/pages/onboarding.adoc [hunk 2]
```diff
@@ -25,11 +25,9 @@ Picnic introduction, your gear and set-up the office systems. In the
...
+* *Wednesday*: To continue the Picnic distribution experience, you'll spend one full day in one of our Fulfilment Center to be a shopper. You'll get to see how every order is put together, from the moment it is delivered by our supplier, until it's dispatched to a hub.
```
Comment (Stephan202) [inline src/pages/onboarding.adoc]:
```suggestion
* *Wednesday*: To continue the Picnic distribution experience, you'll spend one full day in one of our Fulfilment Centers to be a shopper. You'll get to see how every order is put together, from the moment it is delivered by our supplier, until it's dispatched to a hub.
```

File: src/pages/faq.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
+== Frequently Asked Questions
+
+After reading through, you couldn't find the answer to that one small question you have left? Or maybe you just want to know as much as possible about our process? Worry not! Below, you'll find all sorts of questions that we've been asked in the past (with an answer of course 😉).
+
+=== How do I apply for a job?
+
+Follow the link to our https://join.picnic.app/jobs/[job page] and have a look at our openings. If you find any of them interesting, you can apply at the bottom of every ad. Make sure to give as much detail about yourself as you can, answer the questions (if your job as some) and click `Apply`. Simple!
```
Comment (Stephan202) [inline src/pages/faq.adoc]: "if your job as some" -> What do you mean here? :thinking:

File: src/pages/faq.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+[.text-right]
+=== How long does the process take?
+
+The answer is dependent on a lot of factor, like your availability, how quickly you submit your assignment and where you live. On average, we are able to go from application to offer in a month. Not to brag though, but our record is 6 days. 🏆
```
Comment (Stephan202) [inline src/pages/faq.adoc]:
```suggestion
The answer is dependent on a lot of factors, like your availability, how quickly you submit your assignment and where you live. On average, we are able to go from application to offer in a month. Not to brag though, but our record is 6 days. 🏆
```

File: src/pages/faq.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+[.text-left]
+=== What should I wear?
+
+Whatever you feel like. We don't have a dresscode (whether for interviews or on a daily basis) so make sure you wear what you like. Our only rule is that we all should wear something...
+
+[.text-right]
+=== I need to travel to Amsterdam for the on-site vist? Are the costs covered?
+
+Yes! We take care of bookking your flights and hotel room and cover those costs. As part of the visit, you will most likely have lunch with us, however, meals (along with anythins else) taking place outside the office are on you.
```
Comment (Stephan202) [inline src/pages/faq.adoc]:
```suggestion
Yes! We take care of booking your flights and hotel room and cover those costs. As part of the visit you will most likely have lunch with us. However, meals (along with anything else) taking place outside the office are on you.
```

File: src/pages/faq.adoc [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+[.text-left]
+=== What about the moving costs?
+
+When you move to Amsterdam to work at Picnic, we cover the costs of your flights (for you and your difrect family) and we put you up in one of the Picnic apartments (or hotel room should there be no apartment available) for one month, to give you time to find your own place while getting onboarded.
```
Comment (Stephan202) [inline src/pages/faq.adoc]:
```suggestion
When you move to Amsterdam to work at Picnic, we cover the costs of your flights (for you and your direct family) and we put you up in one of the Picnic apartments (or hotel room should there be no apartment available) for one month, to give you time to find your own place while getting onboarded. 
```

File: src/pages/faq.adoc (hunk 4 above)
Comment (Mordavolt) [inline src/pages/faq.adoc]:
```suggestion
The answer is dependent on a lot of factors, like your availability, how quickly you submit your assignment and where you live. On average, we are able to go from application to offer in a month. Not to brag though, but our record is 6 days. 🏆
```

File: src/pages/faq.adoc (hunk 5 above)
Comment (Mordavolt) [inline src/pages/faq.adoc]:
```suggestion
Yes! We take care of booking your flights and hotel room and cover those costs. As part of the visit, you will most likely have lunch with us, however, meals (along with anything else) taking place outside the office are on you.
```

**Review by Mordavolt — COMMENTED** [review]
Nice, some typos though. Would be nice to incorporate some proofreader in the process.

**Comment by WillemGerritzen** [comment]
Added [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) to my IDE, should reduce typos.

**Review by Stephan202 — COMMENTED** [review]
The `src/pages/onboarding.adoc` are yet to be addressed, but the rest looks much better now :muscle:

**Review by Stephan202 — APPROVED** [review]

**Pull request merged**